CODEOWNERS

```diff
@@ -9,20 +9,13 @@
 # will be requested to review.
 
 # include files
-src/include/sof/dmic.h			@singalsu
-src/include/host/*			@ranj063
 src/include/ipc/**			@thesofproject/steering-committee
 src/include/kernel/**			@thesofproject/steering-committee
 src/include/user/**			@thesofproject/steering-committee
-src/include/gdb/*			@mrajwa
-src/include/audio/kpb.h			@mrajwa
-src/include/audio/mux.h			@akloniex
 
 # audio component
 src/audio/src*				@singalsu
 src/audio/eq*				@singalsu
-src/audio/fir*				@singalsu
-src/audio/iir*				@singalsu
```

Comment:
I think it's better to replace with src/audio/eq_* instead of removing. Same for similar cases.

Reply:
@jajanusz, @singalsu , others can you please explicit the "similar cases"? I'm new to this codebase.

(why this is a github Draft for now)